.prettierignore

```diff
@@ -0,0 +1,2 @@
+# ignore markdown files so we can format them as we want
+.md
```

Comment:
Are you sure about this? Markdown is user-facing, and having at least a modicum of formatting does not look like a bad idea. Is there anything specific that made you include this?